[2024-04-13T01:28:32.860+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 12345678.insert_qdrant manual__2024-04-12T20:34:09.777728+00:00 [queued]>
[2024-04-13T01:28:32.866+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 12345678.insert_qdrant manual__2024-04-12T20:34:09.777728+00:00 [queued]>
[2024-04-13T01:28:32.866+0000] {taskinstance.py:2193} INFO - Starting attempt 2 of 3
[2024-04-13T01:28:32.874+0000] {taskinstance.py:2214} INFO - Executing <Task(PythonOperator): insert_qdrant> on 2024-04-12 20:34:09.777728+00:00
[2024-04-13T01:28:32.878+0000] {standard_task_runner.py:60} INFO - Started process 2974 to run task
[2024-04-13T01:28:32.881+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', '12345678', 'insert_qdrant', 'manual__2024-04-12T20:34:09.777728+00:00', '--job-id', '109', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpppoxvq80']
[2024-04-13T01:28:32.883+0000] {standard_task_runner.py:88} INFO - Job 109: Subtask insert_qdrant
[2024-04-13T01:28:32.914+0000] {task_command.py:423} INFO - Running <TaskInstance: 12345678.insert_qdrant manual__2024-04-12T20:34:09.777728+00:00 [running]> on host 5feba290cccc
[2024-04-13T01:28:32.969+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='12345678' AIRFLOW_CTX_DAG_ID='12345678' AIRFLOW_CTX_TASK_ID='insert_qdrant' AIRFLOW_CTX_EXECUTION_DATE='2024-04-12T20:34:09.777728+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-12T20:34:09.777728+00:00'
[2024-04-13T01:28:32.980+0000] {python.py:202} INFO - Done. Returned value was: {'status': 'success', 'message': 'Data inserted to Vector DB successfully'}
[2024-04-13T01:28:32.993+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=12345678, task_id=insert_qdrant, execution_date=20240412T203409, start_date=20240413T012832, end_date=20240413T012832
[2024-04-13T01:28:33.022+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-04-13T01:28:33.041+0000] {taskinstance.py:3309} INFO - 1 downstream tasks scheduled from follow-on schedule check
